### ISSUE TITLE
[Quest][Classic] Hinterlands Horde Quest: Recover the Key! Endlessly being given out

### ISSUE THREAD

**Issue by shimonsays** [dalaranwow/dalaran-wow#5351]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The Hinterlands Horde quest "Recover the Key!" or the quest giver (Elder Torntusk) is bugged at the moment. Instead of "Recover the Key!" normally concluding and proceeding onto the next quest in the chain ("Return to Primal Torntusk"), the quest seems to hang on some coding path and not have the quest giver Elder Torntusk give the next quest. Instead, Elder Torntusk will give "Recover the Key!" again, which gives the reward of experience again.

**Expected behaviour**: Quest completes normally and proceeds to the next in chain ("Return to Primal Torntusk")

**Steps to reproduce the problem**:

1. Complete "Recover the Key!"
2. Turn in the quest in an attempt to get the next quest in the chain ("Return to Primal Torntusk")
3. Receive "Recover the Key!" again instead of the next quest.
4. Complete "Recover the Key!" again. Notice additional Exp or Gold at max level.

**Include proofs for this behaviour**

https://classicdb.ch/?quest=7846

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Quest ID: 7846
Quest-giver NPC: 14757

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
issue confirmed, the questchain is wrong

**Comment by Rushor** [comment]
oh duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3893
